app/cmd/root.go

```diff
@@ -62,7 +58,11 @@ func init() {
 }
 
 func initConfig() {
-	if rootOptions.ConfigFile == "" {
+	fmt.Println("Jenkins CLI (jcli) manage your Jenkins")
+	if rootOptions.Version {
+		fmt.Printf("Version: %s\n", app.GetVersion())
```

Comment:
Method `initConfig` is depended on all other methods. Usually, we print the version when executing the root cmd.